src/apis/content.api.ts

```diff
@@ -0,0 +1,46 @@
+import { IContent } from '@/types/content';
+import { httpClient } from './http.api';
+
+export interface FetchContentParams {
+  content_id: string | undefined;
+}
```

Comment:
```suggestion
export interface FetchContentParams {
  content_id?: string;
}
```

undefined가 올 수 있는 경우에는 ?를 이용하여 표시하기도 한답니다.
근데, 단 하나의 인자만 온다면 굳이 interface로 선언할 필요가 있을까요? 🤔 
type으로 선언해도 충분할 것 같기도 합니다.
그렇다면 ?는 못쓰겠지만요 😅